lib/smack/DSAWrapper.cpp

```diff
@@ -215,23 +118,115 @@ unsigned DSAWrapper::getPointedTypeSize(const Value *v) {
 }
 
 int DSAWrapper::getOffset(const Value *v) {
-  return getOffset(new MemoryLocation(v));
+  if (DG->hasCell(*v)) {
+    auto cell = DG->getCell(*v);
```

Comment:
Add `assert(cell)` after this line as a sanity check.

Reply:
Done.

Reply:
> Add `assert(cell)` after this line as a sanity check.

`cell` is a reference not a pointer.